_integration_tests/test/user_tasks/get_waiting_for_process_model.js

```diff
@@ -82,6 +82,11 @@ describe('Consumer API:   GET  ->  /process_models/:process_model_id/userTasks',
     should(formField).have.property('enumValues');
     should(formField).have.property('label');
     should(formField).have.property('defaultValue');
+
+    should(userTask.data).have.property('description');
+    should(userTask.data).have.property('finishedMessage');
+    should(userTask.data.description).is.eql('TestDescription');
```

Comment:
```suggestion
    should(userTask.data.description).be.eql('TestDescription');
```

This way, the assertion reads like an actual sentence.